app.js

```diff
@@ -30,7 +30,8 @@ app.use(
 app.get('/health', (req, res) => {
   res.json({ ok: true });
 });
-app.get('/', swaggerUi.serve, swaggerUi.setup(swaggerDocument));
+app.use('/', swaggerUi.serve);
+app.get('/', swaggerUi.setup(swaggerDocument));
```

Comment:
you can add them in one handler like this
`app.use('/', swaggerUi.serve, swaggerUi.setup(swaggerDocument));` it didn't work with you initially because you added `app.get` instead of `app.use`.